### PR TITLE
Add descriptions to the Alibaba integration parameters

### PR DESCRIPTION
**Pull request by ShirleyDenkberg**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-hq.paloaltonetworks.local/browse/CRTX-67257

## Description
Provide descriptions for the integration parameters.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/AlibabaActionTrail/Integrations/AlibabaActionTrailEventCollector/README.md [hunk 1]
```diff
@@ -7,18 +7,18 @@ This integration was integrated and tested with API version 0.6 of Alicloud Log
 2. Search for Alibaba Action Trail Event Collector.
 3. Click **Add instance** to create and configure a new integration instance.
```
Comment (nkanon) [inline Packs/AlibabaActionTrail/Integrations/AlibabaActionTrailEventCollector/README.md:9]: Hi @ShirleyDenkberg,
Item 1 is not correct as the pack is located somewhere else.
It should be:
1. Navigate to Settings > Configurations > Automation & Feed Integrations

File: Packs/AlibabaActionTrail/Integrations/AlibabaActionTrailEventCollector/README.md [hunk 2]
```diff
@@ -7,18 +7,18 @@ This integration was integrated and tested with API version 0.6 of Alicloud Log
...
+    | --- | --- | --- |
+    | Endpoint | The URL used to access your project and the data of your project. |True |
+    | Access key id | The ID  used to identify the user. | True |
+    | Access key | The key provided to you by Alibaba Cloud for authentication. | True |
+    | Project name | The name of your project in your Log Service used to isolate the resources of different users and control access to specific resources. | True |
+    | Logstore name | The unit in your Log Service that is used to collect, store, and query logs. | True |
+    | Query | The filter conditions in search statements used to obtain specific logs. Each query statement consists of a search statement and an analytic statement. The search statement and the analytic statement are separated with a vertical bar (\|).A search statement can be a keyword, a numeric value, a numeric value range, a space, or an asterisk . If you specify a space or an asterisk  as the search statement, no conditions are used for searching, and all logs are returned. <br />For example: * \| SELECT status, count AS PV GROUP BY status, will return all logs based on the analytic statement. | True |
```
Comment (nkanon) [inline Packs/AlibabaActionTrail/Integrations/AlibabaActionTrailEventCollector/README.md:17]: Hi, I think we should put as an example the default query that is set for the integration.
Query: *| select * from actiontrail_pa_trail
Description: This query retrieves all the events from the project set above.

**Comment by ShirleyDenkberg** [comment]
@NadavNuss Thanks.  Your comments were perfect.

**Comment by nkanon** [comment]
> @NadavNuss Thanks. Your comments were perfect.

Hi, You tagged his by mistake :)

**Comment by ShirleyDenkberg** [comment]
@nkanon Oops.  I edited the comment.  Can you approve this PR so we can close it?

**Review by nkanon — APPROVED** [review]

**Comment by richardbluestone** [comment]
As discussed this is not relevant for XSOAR but XSIAM. My comments are referring to https://xsoar.pan.dev/docs/reference/integrations/alibaba-action-trail-event-collector 
1. It refers to supported versions as XSOAR 6.8.0. I'm not sure how this is generated but it should be XSIAM.
2. Alibaba logs event collector integration for XSIAM. It should say Alibaba log event collector integration for XSIAM.
3. The commands section refers to XSOAR. You can run this in the Alert War Room in the CLI in XSIAM. 
Do you want me to make changes? Although I don't know about 1 above.

**Comment by ShirleyDenkberg** [comment]
@richardbluestone 
I implemented comments #2 
       Alibaba log event collector integration for XSIAM.
and #3. 
      You can execute these commands Alert War Room in the CLI in XSIAM.
      After you successfully execute a command, a DBot message appears in the War Room with the command details.

@nkanon Referring to Richard's first comment.  I do not know how to update this.  Can you help? This appears in Pan Dev, but it does not appear in the md file.
       It refers to supported versions as XSOAR 6.8.0. I'm not sure how this is generated but it should be XSIAM.

**Pull request closed without merging**